types/src/on_chain_config/execution_config.rs

```diff
@@ -103,13 +103,13 @@ impl OnChainExecutionConfig {
 impl BlockGasLimitType {
     pub fn default_for_genesis() -> Self {
         BlockGasLimitType::ComplexLimitV1 {
-            effective_block_gas_limit: 30000,
+            effective_block_gas_limit: 20000,
             execution_gas_effective_multiplier: 1,
             io_gas_effective_multiplier: 1,
             conflict_penalty_window: 9,
             use_granular_resource_group_conflicts: false,
             use_module_publishing_block_conflict: true,
-            block_output_limit: Some(5 * 1024 * 1024),
+            block_output_limit: Some(4 * 1024 * 1024),
```

Comment:
how did you get these numbers?

Reply:
mostly i changed it proportionally with respect to new block rate and rounded up, assuming it was well calibrated before. i will take a look at a forge stable run as well and expect to tune again if necessary.